docs/external_plugins/introduction.md

```diff
@@ -8,14 +8,11 @@ External plugins let Wash talk to other things outside of the built-in plugins.
 
 2. Write the plugin script. This is the script that Wash will shell out to whenever it needs to invoke an action on a specific entry within your plugin.
 
-3. Add the plugin to the (configurable) `plugins.yaml` file by specifying its name and path to the plugin script. An example `plugins.yaml` file is shown below:
+3. Add the plugin to the (configurable) `plugins.yaml` file by specifying a path to the plugin script. The name will be determined by invoking the script with the `init` action. An example `plugins.yaml` file is shown below:
 
     ```
-    - name: 'external-aws'
-      script: '/Users/enis.inan/wash/external-plugins/external-aws.rb'
-  
-    - name: 'network'
-      script: '/Users/enis.inan/wash/external-plugins/network.sh'
+    - script: '/Users/enis.inan/wash/external-plugins/external-aws.rb'
```

Comment:
Should we keep Name just for logging output? Should we keep this an object with a single field, or accept a list of scripts?